src/SFA.DAS.EmployerIncentives.Database/Tables/Payment.sql

```diff
@@ -13,6 +13,7 @@
 	[Amount] DECIMAL(9, 2) NOT NULL
 )
 GO
-CREATE INDEX IX_Payment_ApprenticeshipIncentiveId ON [incentives].[Payment] (ApprenticeshipIncentiveId)
+CREATE CLUSTERED INDEX IX_Payment_PaidDate ON [incentives].[Payment] (PaidDate)
```

Comment:
I'm not convinced this index will be used so it'd be good to check the execution plan.  It certainly shouldn't be used when we have the IDs.